build.gradle

```diff
@@ -1,113 +1,113 @@
-                pom {
-
-                    withXml {
-                        def dependenciesNode = asNode().appendNode('dependencies')
-
-                        configurations.api.dependencies.each {
-                            if (it.name == 'unspecified')
-                                return
-                            def dependencyNode = dependenciesNode.appendNode('dependency')
-                            dependencyNode.appendNode('groupId', it.group)
-                            dependencyNode.appendNode('artifactId', it.name)
-                            dependencyNode.appendNode('version', it.version)
-                            dependencyNode.appendNode('scope', 'compile')
-                        }
-
-                        configurations.implementation.dependencies.each {
-                            if (it.name == 'unspecified')
-                                return
-                            def dependencyNode = dependenciesNode.appendNode('dependency')
-                            dependencyNode.appendNode('groupId', it.group)
-                            dependencyNode.appendNode('artifactId', it.name)
-                            dependencyNode.appendNode('version', it.version)
-                            dependencyNode.appendNode('scope', 'runtime')
-                        }
-                    }
-                    //noinspection GroovyAssignabilityCheck
-                    project {
-                        name ARTIFACT_ID
-                        description PROJECT_DESCRIPTION
-                        url SITE_URL
-                        inceptionYear 2016
-
-                        packaging 'aar'
-                        artifactId ARTIFACT_ID
-                        version VERSION_NAME
-
-                        // Set your license
-                        licenses {
-                            license {
-                                name 'The Apache Software License, Version 2.0'
-                                url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
-                                distribution 'repo'
-                            }
-                        }
-                        developers {
-                            developer {
-                                id 'olssonhenrik'
-                                name 'Henrik Olsson'
-                                email 'henrikolsson@outlook.com'
-                            }
-                        }
-                        scm {
-                            connection GIT_URL
-                            developerConnection GIT_URL
-                            url SITE_URL
-                        }
-                    }
-                }.writeTo("build/publications/release/pom-default.xml")
-            }
-        }
-    }
-}
-
-task clean(type: Delete) {
-    delete rootProject.buildDir
-}
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
```

Comment:
Whitespace in this file should be kept to reduce the diff.

Reply:
I'm really not sure what happened here. The whitespace seems the same to me, maybe line endings changed?

Reply:
Probably. A meaningful editor should give a clue ;-)

Reply:
Ok, now it should be good to go.